otterdog/jetty.jsonnet

```diff
@@ -131,20 +131,50 @@ orgs.newOrg('jetty') {
       ],
       branch_protection_rules: [
         orgs.newBranchProtectionRule('jetty-9.4.x') {
+          bypass_pull_request_allowances+: [
+            "@gregw",
```

Comment:
you could specify there @jetty/rt-jetty-committers to include all committers to the list of bypassers.

Reply:
We used to only allow the PMC members which is essentially the list you see.

The skillset of this group is sufficient to trust to with this ability.

Reply:
ok then the @jetty/rt-jetty-project-leads group is the right one.